OpenRA.Mods.Common/Projectiles/Bullet.cs

```diff
@@ -233,6 +236,7 @@ public void Tick(World world)
 				length = Math.Max((target - pos).Length / speed.Length, 1);
 				ticks = 0;
 				source = pos;
+				Game.Sound.Play(SoundType.World, info.BounceSound, source);
```

Comment:
```suggestion
			if (info.BounceSound != null)
				Game.Sound.Play(SoundType.World, info.BounceSound, source);
```

Reply:
I don't think a null check here is strictly necessary? https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Game/Sound/Sound.cs#L116-L117

Reply:
You are right. It is only required when submitting a `string[]`.